contracts/src/v0.8/vrf/dev/VRFV2PlusWrapper.sol

```diff
@@ -148,11 +156,17 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     if (_linkNativeFeed != address(0)) {
       s_linkNativeFeed = AggregatorV3Interface(_linkNativeFeed);
     }
+    if (_subId == 0) {
+      revert SubscriptionIdMissing();
+    }
+
+    // Sanity check: should revert if the subscription does not exist
+    s_vrfCoordinator.getSubscription(_subId);
 
-    // Create this wrapper's subscription and add itself as a consumer.
-    uint256 subId = s_vrfCoordinator.createSubscription();
-    SUBSCRIPTION_ID = subId;
-    s_vrfCoordinator.addConsumer(subId, address(this));
+    // Subscription for the wrapper is created and managed by an external EOA
+    // Expectation is that wrapper contract address will be added as a consumer
+    // to this subscription by the external EOA (owner of the subscription)
```

Comment:
instead of EOA, should we say external account? sub owner will most likely be an external contract or multi sig safe, which is also a contract, not an EOA

Reply:
Makes sense